application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java

```diff
@@ -363,23 +363,25 @@ private void executeNoActivityAdviceCheck(long threadChannelId, long authorId) {
                     threadChannelId, authorId);
             return;
         }
-        if (threadChannel.getHistory()
-            .getRetrievedHistory()
-            .stream()
-            .map(Message::getAuthor)
-            .map(User::getIdLong)
-            .anyMatch(id -> id == authorId)) {
+        if (noAuthorActivity(authorId, threadChannel)) {
             MessageEmbed embed = HelpSystemHelper.embedWith(
                     """
-                            Hey there {}👋 It has been five minutes and you have not shown any activity in this thread.
-                            Please elaborate on your question, or close the thread if it was created accidentally, thanks 🙂
-                            """,
-                    User.fromId(authorId).getAsMention());
-            MessageCreateData message = new MessageCreateBuilder().setEmbeds(embed).build();
-            threadChannel.sendMessage(message).queue();
-
+                            Hey there %s👋  It has been a bit after you created this thread and you still did not share any details of your question.
+                            Helpers have seen your question already and are just waiting for you to elaborate on your problem and provide detailed information on it 👌
+                            If this thread was created accidentally, you can close it.
+                            """
+                        .formatted(User.fromId(authorId).getAsMention()));
+            threadChannel.sendMessageEmbeds(embed).queue();
         }
+    }
 
+    private static boolean noAuthorActivity(long authorId, ThreadChannel threadChannel) {
```

Comment:
should start with a verb like `hasNoAuthorActivity`

Reply:
Done